com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -614,6 +631,23 @@ void UpdateInfo(int stateIndex, RunItemContext ctx)
 
         void Convert(ClickEvent evt)
         {
+            // Ask to save save the current open scene and after the conversion is done reload the same scene.
+            Scene currentScene = SceneManager.GetActiveScene();
+            string currentScenePath = currentScene.path;
```

Comment:
Seems like getting the path is only used in lines 694 and 696. 
If you extract this scene dirty check in to a function, then you can move the line for the path down there.